### PR TITLE
feat: ID-19/Create-key-folder-layers

### PR DESCRIPTION
**Pull request by salet-gutierrez-jalafund**
## Trello Ticket

[ID-19](https://trello.com/c/GC2Bkxyz)

## What did I do?
What I did was add `remove_later.txt` files so that you can see the basic structure of the folders that will be in the program, which will be modified.

## Why did I do this?
To have a basic structure on the folders and so that all the classes that will be added are organized.

## How did I do it?
I created folders and .txt files

## Note
This is just a basic folder structure, later in the project it will be modified either by deleting it or adding more of these.

### CODE REVIEW

**Comment by salet-gutierrez-jalafund** [comment]
[ID-19: Project Initialization](https://trello.com/c/GC2Bkxyz/27-id-19-project-initialization)

File: src/ExpenseTrackerGrupo2/Program.cs
Comment (alex-paca-jalafund) [inline src/ExpenseTrackerGrupo2/Program.cs:1]: This direction src/ExpenseTrackerGrupo2/ its in use on Dockerfile configuration

**Review by alex-paca-jalafund — CHANGES_REQUESTED** [review]

File: src/ExpenseTrackerGrupo2/Program.cs
Comment (salet-gutierrez-jalafund) [reply src/ExpenseTrackerGrupo2/Program.cs:1]: Done

File: src/DataAccess/Entities/remove_later.txt
Comment (alex-paca-jalafund) [inline src/DataAccess/Entities/remove_later.txt:1]: in dockerfile the workspace its setted for src/ExpenseTrackerGrupo2 so all the directories has to be inside the workspace:

# Build
COPY ["ExpenseTrackerGrupo2", "ExpenseTrackerGrupo2/"]
WORKDIR /src/ExpenseTrackerGrupo2
RUN dotnet build "ExpenseTrackerGrupo2.csproj" -c Release -o /app/build

**Review by alex-paca-jalafund — CHANGES_REQUESTED** [review]

**Comment by salet-gutierrez-jalafund** [comment]
For reasons that joint tasks are being carried out in this branch and the name is not being met, this branch will be closed, which will open another one to carry out a CI hotfix and a feature for the architecture of the branches.

**Pull request closed without merging**